src/components/navigation/topnav.tsx

```
@@ -1,15 +1,177 @@
+  }, [navigation, isNavLoading]);
+
+  const SaveButtonIcon = () => {
+    switch (saveState) {
+      case 'saving':
+        return <Loader2 size={16} className="animate-spin" />
+      case 'saved':
+        return <Check size={16} />
+      case 'error':
+        return <AlertCircle size={16} />
+      case 'idle':
+      default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)